### PR TITLE
[textanalytics] Moves textanalytics track2 and renames namespace/package

### PR DESCRIPTION
**Pull request by kristapratico**
- Moves Text Analytics Track 2 to `sdk/textanalytics`
- Renames package/namespace to `azure-ai-textanalytics`
- Adds namespace package `azure-ai-nspkg`
- Rename changes from this PR #9094 
- Updates generated code due to fixes in the swagger

Link to text analytics tests CI: [pass](https://dev.azure.com/azure-sdk/playground/_build/results?buildId=213069&view=results)

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/textanalytics/azure-ai-nspkg/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Since version 3.0, this is Python 2 package only, Python 3.x SDKs will use `PEP420 <https://www.python.org/dev/peps/pep-0420/>` as namespace package strategy.
+To avoid issues with package servers that does not support `python_requires`, a Python 3 package is installed but is empty.
+
+It provides the necessary files for other packages to extend the azure.ai namespace.
+
+If you are looking to install the Azure client libraries, see the
+`azure <https://pypi.python.org/pypi/azure>`__ bundle package.
+
+
+![Impressions](https://azure-sdk-impressions.azurewebsites.net/api/impressions/azure-sdk-for-python%2Fsdk%2Ftextanalytics%2Fazure-ai-nspkg%2FFREADME.png)
```
[the remark this answers is not in the document]
Comment (kristapratico) [reply sdk/textanalytics/azure-ai-nspkg/README.md]: I think the extra `F` is a copy/paste mistake. Most of the READMEs I looked at did not have it so I removed it.

File: sdk/textanalytics/azure-ai-nspkg/setup.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    author='Microsoft Corporation',
+    author_email='azpysdkhelp@microsoft.com',
+    url='https://github.com/Azure/azure-sdk-for-python',
+    classifiers=[
+        'Development Status :: 5 - Production/Stable',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.4',
```
[the remark this answers is not in the document]
Comment (kristapratico) [reply sdk/textanalytics/azure-ai-nspkg/setup.py]: Removed

**Comment by kristapratico** [comment]
This PR just moves Track 2 to its new directory. I already reverted the PR that overwrote track 1 with track 2 in the cognitiveservices directory.

**Pull request merged**